apps/mobile/app/db/init.ts

```diff
@@ -1,20 +1,39 @@
 import * as SQLite from "expo-sqlite";
+import { createSingletonDbProvider } from "../sync/SyncedExpoDB";
+import { cryb64 } from "@vlcn.io/ws-common";
+import {decode, encode} from 'base-64'
+if (!global.btoa) {
+  global.btoa = encode;
+}
+
+if (!global.atob) {
+  global.atob = decode;
+}
 
-export const db = SQLite.openDatabase("test.db");
+export const dbName = "test.db";
+export const db = SQLite.openDatabase(dbName);
+
+// TODO: ideally we can share the schema in a package between client and server
+// this is curently duplicated into /server/schemas/test.sql
+const schema = [
+  `CREATE TABLE IF NOT EXISTS "todo" ("id" PRIMARY KEY, "text", "completed" INTEGER DEFAULT 0);`,
+  `SELECT crsql_as_crr('todo');`
+];
```

Comment:
we could create a `shared` directory that contains packages that we share between server and client, and add that to the workspaces list. i can follow up on that in another pr